EndPoints.md

```diff
@@ -19,14 +21,14 @@ get -> /ui/icon -> view icon
 
 post -> /ui/icon -> save icon
 
-put -> /ui/icon/id -> update icon
+put -> /ui/icon -> update icon
```

Comment:
Why you delete the id? remember : the JWT only provides the user info. We dont have access to the icon id that needs to update